docs/source/examples/workspace/linchpin.conf

```diff
@@ -91,6 +91,11 @@ distill_data = True
 # data upon failure).
 distill_on_error = False
 
+# User can make linchpin use the actual return codes for final return code
+# if enabled True, even if one target provision is successfull linchpin 
+# returns exit code zero else returns the sum of all the return codes
+use_actual_rcs = False
+
```

Comment:
Are you overriding what is in linchpin.constants here? It appears not, so it would be prudent to just comment it out.